Cargo.toml

```diff
@@ -30,6 +30,7 @@ wasmtime-wast = { workspace = true }
 wasmtime-wasi = { workspace = true }
 wasmtime-wasi-crypto = { workspace = true, optional = true }
 wasmtime-wasi-nn = { workspace = true, optional = true }
+wasmtime-runtime = { workspace = true }
```

Comment:
This was only needed for MmapVec so I'll remove on the next commit. (see [below](https://github.com/bytecodealliance/wasmtime/pull/5035#discussion_r992230460).)

Reply:
After re-reading my previous commits and review comments here, I realized this reply is not possible at all. MmapVec cannot be avoided because `load_module` should do both "header check" and "copyless mmap" in one path. I think your suggestion is truly valid but please understand that I don't know much about this project (yet) to implement it rapidly so I would come back with results after some time with some investigation and tryouts.

Reply:
Sounds good. And thanks for working on this! I agree that running modules from stdin is an interesting idea.